buildspecs/runBenchmarks.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+cd test/sdk-benchmarks
+BENCH_MARKS=()
+if git diff --name-only master HEAD | grep -q "http-clients/"; then
+    echo "Found changes in http-clients module, will run ApiCallHttpClientBenchmark and ClientCreationBenchmark"
+    BENCH_MARKS+=(ApiCallHttpClientBenchmark.run.UrlConnectionHttpClient:$URL_CONNECTION_CLIENT_BASELINE_SCORE)
+    BENCH_MARKS+=(ApiCallHttpClientBenchmark.run.ApacheHttpClient:$APACHE_CLIENT_BASELINE_SCORE)
+fi
+
+if git diff --name-only master HEAD | grep -q "core/"; then
```

Comment:
This can be shortened with `git diff master core`

Reply:
Believe we should also be using `--exit-code` here:

> Make the program exit with codes similar to diff(1). That is, it exits with 1 if there were differences and 0 means no differences.

Reply:
+1